leetcode/client.go

```diff
@@ -95,7 +95,7 @@ func NewClient(options ...ClientOption) Client {
 	httpClient := sling.New()
 	httpClient.Add(
 		"User-Agent",
-		"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.55 Safari/537.36",
+		"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.55 Safari/537.36 Edg/109.0.1518.78",
```

Comment:
no need to change the UA I think ?

Reply:
I added it to prevent anomalies.

Reply:
> no need to change the UA I think ?

I'll test this one later.

Reply:
> no need to change the UA I think ?

yep,It's work. I have removed  `Edg/109.0.1518.78`